packages/protocol/contracts/governance/Validators.sol

```diff
@@ -269,12 +364,15 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
     // Use the proof of possession bytes
     require(checkProofOfPossession(publicKeysData.slice(64, 48 + 96)));
 
-    address account = getLockedGold().getAccountFromValidator(msg.sender);
+    address account = getLockedGold().getAccountFromActiveValidator(msg.sender);
```

Comment:
if we keep this `active` vs `inactive` validator, i would make the default method (without prefix) for the active. So `getAccountFromValidator()` or even better `validatorToAccount()` :P; the method for the current validator and some other method with prefix `Inactive`, `Old`... for the other.

Reason? It's easier to forget (for a future developer) that the `getAccountFromValidator()` refers also to the inactive ones

Reply:
maybe use word "history"? Like "search for the account whose signing validator key is/has been this in their *history*"? `getAccountFromValidatorHistory`? 
idk, english is your native language :P